client/src/components/library/admin/BookForm.js

```diff
@@ -21,7 +21,7 @@ class BookForm extends Component {
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  onChange (e){
+  onChange(e) {
     if (!this.state.errors[e.target.name]) {
       const errors = Object.assign({}, this.state.errors);
       delete errors[e.target.name];
```

Comment:
Why is this happening?

Reply:
to ensure an error is not populated again for a field if there are no errors after correction